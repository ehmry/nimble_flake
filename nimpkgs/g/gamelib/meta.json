{
  "alias": null,
  "description": "A library of functions to make creating games using Nim and SDL2 easier. This does not intend to be a full blown engine and tries to keep all the components loosely coupled so that individual parts can be used separately.",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "gamelib",
  "tags": [
    "sdl",
    "game",
    "library"
  ],
  "url": "https://github.com/PMunch/SDLGamelib",
  "web": "https://github.com/PMunch/SDLGamelib"
}
