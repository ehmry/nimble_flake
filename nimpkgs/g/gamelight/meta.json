{
  "alias": null,
  "description": "A set of simple modules for writing a JavaScript 2D game.",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "gamelight",
  "tags": [
    "js",
    "library",
    "graphics",
    "collision",
    "2d"
  ],
  "url": "https://github.com/dom96/gamelight",
  "web": "https://github.com/dom96/gamelight"
}
