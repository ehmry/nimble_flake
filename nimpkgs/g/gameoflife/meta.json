{
  "alias": null,
  "description": "gameoflife is library for Game of Life.",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "gameoflife",
  "tags": [
    "gameoflife",
    "library"
  ],
  "url": "https://github.com/jiro4989/gameoflife",
  "web": "https://github.com/jiro4989/gameoflife"
}
