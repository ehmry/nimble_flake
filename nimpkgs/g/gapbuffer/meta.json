{
  "alias": null,
  "description": "A simple gap buffer implementation",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "gapbuffer",
  "tags": [
    "buffer",
    "seq",
    "sequence",
    "string",
    "gapbuffer"
  ],
  "url": "https://notabug.org/vktec/nim-gapbuffer.git",
  "web": "https://notabug.org/vktec/nim-gapbuffer"
}
