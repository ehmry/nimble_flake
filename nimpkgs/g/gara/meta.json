{
  "alias": null,
  "description": "A pattern matching library",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "gara",
  "tags": [
    "nim",
    "pattern"
  ],
  "url": "https://github.com/alehander42/gara",
  "web": "https://github.com/alehander42/gara"
}
