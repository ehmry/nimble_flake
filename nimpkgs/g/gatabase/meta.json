{
  "alias": null,
  "description": "Postgres Database ORM for Nim",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "gatabase",
  "tags": [
    "database",
    "orm",
    "postgres",
    "sql"
  ],
  "url": "https://github.com/juancarlospaco/nim-gatabase",
  "web": "https://github.com/juancarlospaco/nim-gatabase"
}
