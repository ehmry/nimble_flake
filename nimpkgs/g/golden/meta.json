{
  "alias": null,
  "description": "a benchmark tool",
  "doc": null,
  "license": "MIT",
  "method": "git",
  "name": "golden",
  "tags": [
    "benchmark",
    "profile",
    "golden",
    "runtime",
    "run",
    "profiling",
    "bench",
    "speed"
  ],
  "url": "https://github.com/disruptek/golden",
  "web": "https://github.com/disruptek/golden"
}
